omod/src/main/webapp/resources/cdc/csv/wtageinf.csv

```diff
@@ -0,0 +1,77 @@
+Sex,Agemos,L,M,S,P3,P5,P10,P25,P50,P75,P90,P95,P97
+1,0,1.815151075,3.530203168,0.152385273,2.355450986,2.52690402,2.773801848,3.150611082,3.530203168,3.879076559,4.17249339,4.34029274,4.446488308
```

Comment:
What are these csv files used for?

Reply:
these csv files are the percentiles on which the Growth chart depends, and they are the WHO or CDC standards. they  make the graph CDC or WHO standardized